src/transformers/image_transforms.py

```diff
@@ -458,11 +458,17 @@ def center_crop(
     crop_height, crop_width = size
     crop_height, crop_width = int(crop_height), int(crop_width)
 
-    # In case size is odd, (image_shape[0] + size[0]) // 2 won't give the proper result.
-    top = (orig_height - crop_height) // 2
+    # In case height is odd
+    diff_height = orig_height - crop_height
+    if diff_height % 2 == 1:
+        diff_height += 1
+    top = diff_height // 2
```

Comment:
```suggestion
    top = (orig_height - crop_height + 1) // 2
```
would this not work for both odd and non odd?

Reply:
Yep! That would work for odd and even :+1: 
Much cleaner. Thanks for noticing. 
Will apply that. Thanks. :)